assembly/process/sandbox.ts

```diff
@@ -0,0 +1,27 @@
+import { Process } from "./index";
+import { UnmanagedResult } from "../error";
+import { Box, Mailbox } from "../message";
+import { MessageType } from "../message/util";
+
+export class SandboxStartContext<TStart, TReturn> {
```

Comment:
```suggestion
class SandboxStartContext<TStart, TReturn> {
```

Reply:
I don't agree with this, because I never know where I'm going to use the class :)